packages/@glimmer/core/test/interactive/fn-test.ts

```diff
@@ -0,0 +1,149 @@
+// import Component from '@glimmer/component';
```

Comment:
Delete?

Reply:
Ah, actually the intention here is to add these tests back in followups. I commented them out for the time being so tests would pass, but these are still useful tests.

Reply:
Maybe use `todo` then?

Reply:
@chadhietala `QUnit.todo` does not work with testem

Reply:
💩